### PR TITLE
Branch2

### PR DESCRIPTION
**Pull request by AlkaBenawra**
test

### CODE REVIEW

File: decider-starfusion/src/main/java/ca/on/oicr/pde/deciders/STARfusionDecider.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+ * @author alka
+ */
+public class STARfusionDecider extends OicrDecider {
+
+    private String starfusionMemory = "64";
+
+    private Set<String> allowedTemplateTypes;
+    private String queue = "";
+    private String templateType = "WT";
+    private String input_read1_fastq;
```
Comment (prisnir-zz) [inline decider-starfusion/src/main/java/ca/on/oicr/pde/deciders/STARfusionDecider.java]: can you rename variable as inputRead1Fastq, inputRead2Fastq, externalName ?

**Comment by AlkaBenawra** [comment]
retest this please

**Comment by AlkaBenawra** [comment]
retest this please

**Comment by AlkaBenawra** [comment]
retest this please

**Comment by prisnir-zz** [comment]
retest this please

File: decider-starfusion/src/test/resources/development.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="StarDeciderTestSuite" parallel="methods" thread-count="${runTestThreads}">
```
Comment (pruzanov) [inline decider-starfusion/src/test/resources/development.xml:2]: suite name is still StarDeciderTestSuite (Not StarFusionDeciderTestSuite)

File: decider-starfusion/src/test/resources/starfusionTests.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "description": "StarDecider test",
+        "metricsResources": "/expected_output/development/",
+        "parameters": {
+            "--parent-wf-accessions": "538766,996126,5300868,1753095,4186700,5300924,37743"
+            },
+        "iniExclusions": ["parent-accessions", "workflow_run_accession", "workflow-run-accession",
+            "unique_string", "parent_accessions", "parent_accession", "input_file",
+            "input_files", "workflow_bundle_dir", "output_prefix"]
+    },
+    "tests": [
```
Comment (pruzanov) [inline decider-starfusion/src/test/resources/starfusionTests.json:12]: It seems that number of tests doesn't match the number of json files

File: workflow-starfusion/src/main/java/ca/on/oicr/pde/workflows/STARfusionWorkflowClient.java [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+    }
+    
+    private Job copyOutputs(HashMap<String, String> provOut) {
+        List<String> keys = new ArrayList<>(provOut.keySet());
+        Job copyPaths = getWorkflow().createBashJob("copyoutputsjob");      
+        Command cmd = copyPaths.getCommand();
+        for (String k : keys){
+            cmd.addArgument("cp " 
+                    + this.tmpDir + provOut.get(k) + " " 
+                    + this.dataDir + this.outputFilenamePrefix + "." + provOut.get(k) + ";\n");
```
Comment (pruzanov) [inline workflow-starfusion/src/main/java/ca/on/oicr/pde/workflows/STARfusionWorkflowClient.java]: newline escape symbol is not needed

**Comment by prisnir-zz** [comment]
retest this please

**Comment by prisnir-zz** [comment]
retest this please

**Comment by prisnir-zz** [comment]
retest this please

**Pull request merged**